### PR TITLE
Format State Names

### PR DESCRIPTION
**Pull request by SMJ289**
## Type of change made:
- [ ] **Bugfix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [x] **Refactoring** (improves existing functionality)
- [ ] **Testing** (tests existing code or improves existing tests)
- [ ] **Styling** (no functionality changed, just visual improvements)
## Description:

Converts all two letter state abbreviations to full state name and adds state name to search response.

## What functionality does this accomplish?

- Adds a `full_state_name` method to the Formattable module
- Adds additional param to the MemberIndex init method to return the searched state in the response

closes # | does not close anything
## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code
- [x] I have partially tested my code (please explain why): **Still need to add unit testing**

### CODE REVIEW

File: modules/formattable.py [hunk 1]
```diff
@@ -10,3 +10,65 @@ def party_name(party):
...
     else:
       return party
+
+def full_state_name(state_code):
+    states = {
+        'AK': 'Alaska',
+        'AL': 'Alabama',
+        'AR': 'Arkansas',
+        'AS': 'American Samoa',
+        'AZ': 'Arizona',
```
Comment (itemniner) [inline modules/formattable.py:20]: we need to test these methods?

File: app.py [hunk 2]
```diff
@@ -73,9 +73,9 @@ def get_members_by_state(state_):
     senate_results = PropublicaService.senators_by_state(state_)
     house_results = PropublicaService.reps_by_state(state_)
 
-    senator_objects = map(lambda result: MemberIndex(result), senate_results)
+    senator_objects = map(lambda result: MemberIndex(result, state_), senate_results)
 
-    rep_objects = map(lambda result: MemberIndex(result), house_results)
+    rep_objects = map(lambda result: MemberIndex(result, state_), house_results)
```
Comment (itemniner) [inline app.py:78]: great way to pass states!!

File: modules/formattable.py [hunk 3]
```diff
@@ -10,3 +10,65 @@ def party_name(party):
...
+        'UT': 'Utah',
+        'VA': 'Virginia',
+        'VI': 'Virgin Islands',
+        'VT': 'Vermont',
+        'WA': 'Washington',
+        'WI': 'Wisconsin',
+        'WV': 'West Virginia',
+        'WY': 'Wyoming'
+    }
+    return states.get(state_code.upper())
```
Comment (jklich151) [inline modules/formattable.py:74]: Great use of a module to make the code more dry

File: modules/formattable.py (hunk 1 above)
Comment (SMJ289) [reply modules/formattable.py:20]: Yep! We will do that when we add unit testing next.

File: modules/formattable.py (hunk 3 above)
Comment (SMJ289) [reply modules/formattable.py:74]: 🏝️

**Pull request merged**